x-pack/plugins/apm/public/components/app/service_dashboards/index.tsx

```diff
@@ -58,7 +58,7 @@ export function ServiceDashboards() {
     useState<MergedServiceDashboard>();
   const { data: allAvailableDashboards } = useDashboardFetcher();
   const { start, end } = useTimeRange({ rangeFrom, rangeTo });
-  const { dataView } = useApmDataView();
+  const { dataView } = useAdHocApmDataView();
```

Comment:
vs
`const { adHocDataView } = useAdHocApmDataView();`

Reply:
Maybe your suggestion is better, to make this distinction more obvious